exporter/clickhousetracesexporter/config.go

```diff
@@ -19,15 +19,17 @@ import (
 	"go.opentelemetry.io/collector/config"
 )
 
-// Config defines configuration for logging exporter.
+// Config defines configuration for tracing exporter.
 type Config struct {
 	config.ExporterSettings `mapstructure:",squash"` // squash ensures fields are correctly decoded in embedded struct
 
 	Options    `mapstructure:",squash"`
 	Datasource string `mapstructure:"datasource"`
 	Migrations string `mapstructure:"migrations"`
 	// Docker Multi Node Cluster is a flag to enable the docker multi node cluster. Default is false.
-	DockerMultiNodeCluster bool `mapstructure:"docker_multi_node_cluster" default:"false"`
```

Comment:
mapstructure doesn't support setting default value like this so removed it here.
https://github.com/mitchellh/mapstructure/issues/111